setup/upgrade_minor.rst

```diff
@@ -76,12 +82,45 @@ to your code to get everything working. Additionally, some features you're
 using might still work, but might now be deprecated. While that's just fine,
 if you know about these deprecations, you can start to fix them over time.
 
-Every version of Symfony comes with an UPGRADE file (e.g. `UPGRADE-4.1.md`_)
+Every version of Symfony comes with an UPGRADE file (e.g. `UPGRADE-4.4.md`_)
 included in the Symfony directory that describes these changes. If you follow
 the instructions in the document and update your code accordingly, it should be
 safe to update in the future.
 
 These documents can also be found in the `Symfony Repository`_.
 
+.. _updating-flex-recipes:
+
+3) Updating Recipes
+-------------------
+
+Over time - and especially when you upgrade to a new version of a library, an
+updating version of the :ref:`recipe <recipes-description>` may be available.
+These updates are usually minor - e.g. new comments in a configuration file - but
+it's a good idea to update the core Symfony recipes.
+
+Starting in ``symfony/flex`` version 1.6, several commands are available to help
+upgrade your recipes:
+
+.. code-block:: terminal
+
+    # see a list of all installed recipes and which have updates available
+    $ composer recipes
+
+    # see detailed information about a specific recipes
+    $ composer recipes symfony/framework-bundle
+
+    # update a specific recipes
+    $ composer recipes:install symfony/framework-bundle --force -v
+
+The tricky part of this process is that the recipe "update" does not perform
+any intelligent "upgrading" of your code. Instead, **the updates process re-installs
+the latest version of the recipe** which means that **your custom code will be
+overridden completely**. After updating a recipe, you need to carefully choose
+which changes you want, and undo the rest.
```

Comment:
Maybe we should put this before the code block? Or do we assume everyone is using some sort of version control system and no changes will be lost? (I guess many people execute the commands in a code example before reading the next part, at least that's what I often do)